src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -80,7 +74,11 @@ public SnowflakeSinkConnector() {
   @Override
   public void start(final Map<String, String> parsedConfig) {
     Utils.checkConnectorVersion();
-    LOGGER.info(Logging.logMessage("SnowflakeSinkConnector:start"));
+
+    // initialize logging with correlationId
+    LoggerHandler.setCorrelationUuid(UUID.randomUUID());
```

Comment:
This is where the correlationId will be set for all loggers. The goal is to have a new correlationId every time KC starts up